tc-server/src/main/java/com/tc/server/Bootstrap.java

```diff
@@ -103,7 +102,7 @@ private static void writeVersion(ProductInfo info) {
     String longProductString = info.toLongString();
     CONSOLE.info(longProductString);
 
-    CONSOLE.info("Extensions:");
+    CONSOLE.info("Installed extensions:");
```

Comment:
I am also proposing this change because the logs are displaying **ALL** installed extensions through meta-inf/services mechanism. This is not reflecting which extension is actually used at runtime. I.e. some extensions could be disabled through configuration or diagnostic mode.